src/routes.tsx

```diff
@@ -95,6 +96,7 @@ class Routes extends React.Component<AllProps> {
     render() {
         const { registreringstatusData, reaktivertStatus, featureToggles, location } = this.props;
         const erNede = featureToggles['arbeidssokerregistrering.nedetid'];
+        const erNyttaar = featureToggles['arbeidssokerregistrering.nyttaar'];
```

Comment:
Har du tilgang til unleash hjemmfra og får opprettet toggle?

Reply:
ja, tenkte legge lenke til steder, men må legge inn toggles som en del av hva man kan justere på demo først :-)
bilder er på slack btw

Reply:
Er ikke de bildene fra Sketch?

Reply:
skal det være noen forskjell ;-)
Neida, ser jeg bare har postet det ene bildet

Reply:
Ahaaa. Nå så jeg det. Hadde vært kult å se for mobilvisning og! 😊